GVFS/GVFS.Common/Prefetch/BlobPrefetcher.cs

```diff
@@ -162,13 +162,13 @@ public static void AppendToNewlineSeparatedFile(PhysicalFileSystem fileSystem, s
         }
 
         /// <param name="branchOrCommit">A specific branch to filter for, or null for all branches returned from info/refs</param>
-        public virtual void Prefetch(string branchOrCommit, bool isBranch)
+        public virtual void Prefetch(string branchOrCommit, bool isBranch, bool force = false)

```

Comment:
There's no need for a default value for this parameter. There's only one caller of this method, and it passes in a value.